### PR TITLE
ENH add option to create html from notebook using nbconvert

### PR DESCRIPTION
**Pull request by maikia**
#### What does this implement/fix? Explain your changes.
#297 

#### Any other comments?
You are right @rth 
it seems that the jupyter notebook / nbconvert is called every time the toy database is created which seems pointless. I don't think this is a problem with fixture though. However, I am not sure why it has been done and that's why I am a bit hesitant to removing it all together. Do you know what is a reason for it @glemaitre @agramfort ?

in ramp_database-testing.py 
create_toy_db() calls
setup_toy_db() calls 
add_problems() calls 
setup_ramp_kit_ramp_data() calls
subprocess.check_output(["jupyter", "nbconvert", "--to", "html",
                             "{}_starting_kit.ipynb".format(problem_name)])

Jupyter jupyter notebook / nbconvert is also tested in: ramp_utils/deploy.py -> deploy_ramp_event

### CODE REVIEW

**Comment by kegl** [comment]
Because adding the problem means to render the notebook for the problem landing page.

We could get rid of this by not rendering the notebook on the site, just linking the notbook in the github repo. I'd keep it the way it is if it's not a huge problem, I think it's nice to have the description in the web page.

**Comment by agramfort** [comment]
I agree that having the starting kit embedded in the web page is nice and
running nbconvert allows to make sure it's running without crash

>

File: ramp-database/ramp_database/testing.py [hunk 1]
```diff
@@ -123,7 +123,8 @@ def _delete_line_from_file(f_name, line_to_delete):
         f.truncate()
 
 
-def setup_ramp_kit_ramp_data(ramp_config, problem_name, force=False):
+def setup_ramp_kit_ramp_data(ramp_config, problem_name, force=False,
+                            check_notebook=False):
```
Comment (rth) [inline ramp-database/ramp_database/testing.py]: Maybe,
```suggestion
                            convert_notebook=True):
```
and then explicitly set to to false for unit tests? For event deployment I agree that it's nice to have an HTML notebook. Notebook preview on github mostly doesn't work.

**Comment by glemaitre** [comment]
@maikia I think this is fine to have the option which by default does not change the behaviour (as proposed by @rth). However, we are checking that the hml file will exist. So we need to create an empty html file to emulate the conversion. We need a test for this change as well (check that the file is empty).

Do you think that you can finish this PR?

**Comment by glemaitre** [comment]
I will finish this up

**Comment by glemaitre** [comment]
Thanks for the idea @rth. The tests are twice faster.
